frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.module.css

```diff
@@ -57,3 +57,11 @@
   display: flex;
   justify-content: flex-end;
 }
+
+.modalContent {
+  max-height: 60vh !important;
+}
+
+.margin_before_button {
```

Comment:
nit: camel case

Reply:
@andrew032011 the rest of the css file is snake case

Reply:
oh ok! let's leave it